x-pack/filebeat/docs/inputs/input-entity-analytics.asciidoc

```diff
@@ -0,0 +1,116 @@
+[role="xpack"]
+
+:type: entity-analytics
+
+[id="{beatname_lc}-input-{type}"]
+=== Entity Analytics Input
+
+++++
+<titleabbrev>Entity Analytics</titleabbrev>
+++++
+
+Experimental[Input]
+
+The Entity Analytics input collects identity assets, such as users, from external
+identity providers.
+
+The following identity providers are supported:
+
+- <<provider-azure>>
+
+==== Configuration options
+
+The `entity-analytics` input supports the following configuration options plus the
+<<{beatname_lc}-input-{type}-common-options>> described later.
+
+[float]
+==== `provider`
+
+The identity provider. Must be one of: `azure`.
+
+[id="{beatname_lc}-input-{type}-common-options"]
+include::../../../../filebeat/docs/inputs/input-common-options.asciidoc[]
+
+==== Providers
+
+[id="provider-azure"]
+==== Azure Active Directory (`azure`)
```

Comment:
I think a "How it works" section would be useful. Explain what APIs it uses, how it watches for updates, that it enriches with group membership, what it persists, etc.